server/handler.go

```diff
@@ -912,54 +912,51 @@ func (h *Handler) IsLeader() bool {
 
 // PackHistoryHotReadRegions get read hot region info in HistoryHotRegion form.
 func (h *Handler) PackHistoryHotReadRegions() (historyHotRegions []core.HistoryHotRegion, err error) {
-	hotReadLeaderRegions := h.GetHotReadRegions().AsLeader
-	historyLeaderHotRegions, err := h.packHotRegions(hotReadLeaderRegions, true, core.HotRegionTypes[0])
-	if err != nil {
+	hotReadRegions := h.GetHotReadRegions()
+	if hotReadRegions == nil {
 		return
 	}
-	hotReadPeerRegions := h.GetHotReadRegions().AsPeer
-	historyPeerHotRegions, err := h.packHotRegions(hotReadPeerRegions, false, core.HotRegionTypes[0])
+	hotReadPeerRegions := hotReadRegions.AsPeer
+	historyPeerHotRegions, err := h.packHotRegions(hotReadPeerRegions, core.HotRegionTypes[0])
```

Comment:
`core.HotRegionTypes[0]` is not readable, it is better to use enum, like: https://github.com/tikv/pd/blob/20b00a7c4933e307c5423da0c5047250259e19e9/server/statistics/flowkind.go#L17-L24

Reply:
I have two thoughts on this issue
```go
// HotRegionType stands for hot type.
type HotRegionType uint32

// HotRegionType stands for hot type.
var HotRegionTypes = []string{
	WriteType.String(),
	ReadType.String(),
}

// Flags for flow.
const (
	WriteType HotRegionType = iota
	ReadType
)

func (h HotRegionType) String() string {
	switch h {
	case WriteType:
		return "write"
	case ReadType:
		return "read"
	}
	return "unimplemented"
}
```
or
```go
// HotRegionType stands for hot type.
type HotRegionType uint32

// Flags for flow.
const (
	WriteType HotRegionType = iota
	ReadType
)

// HotRegionType stands for hot type.
var HotRegionTypes = []string{
	"write",
	"read",
}
```
this one used in api,we can`t delete this.
```go
var HotRegionTypes = []string{
	"write",
	"read",
}
```

Reply:
I more like the first one.

Reply:
change to first one